documentation/source/users/rmg/database/modification.rst

```diff
@@ -73,6 +72,167 @@ There are several places in the RMG-database and RMG-Py source code where reacti
+Allowed pressure units are ``Pa``, ``bar``, ``atm``, ``torr``, ``psi``, ``mbar``
+(see `PDepArrhenius Class <http://reactionmechanismgenerator.github.io/RMG-Py/reference/kinetics/pdeparrhenius.html>`_ for details)::
+
+	entry(
+	    index = 7,
+	    label = "HCO <=> H + CO",
+	    degeneracy = 1,
+	    kinetics = PDepArrhenius(
+	        pressures = ([1, 10, 20, 50, 100], 'atm'),
+	        arrhenius = [
+	            Arrhenius(A=(9.9e+11, 's^-1'), n=-0.865, Ea=(16755, 'cal/mol'), T0=(1, 'K')),
+	            Arrhenius(A=(7.2e+12, 's^-1'), n=-0.865, Ea=(16755, 'cal/mol'), T0=(1, 'K')),
+	            Arrhenius(A=(1.3e+13, 's^-1'), n=-0.865, Ea=(16755, 'cal/mol'), T0=(1, 'K')),
+	            Arrhenius(A=(2.9e+13, 's^-1'), n=-0.865, Ea=(16755, 'cal/mol'), T0=(1, 'K')),
+	            Arrhenius(A=(5.3e+13, 's^-1'), n=-0.865, Ea=(16755, 'cal/mol'), T0=(1, 'K'))]))
+
+**MultiPDepArrhenius**
+(see `MultiPDepArrhenius Class <http://reactionmechanismgenerator.github.io/RMG-Py/reference/kinetics/multipdeparrhenius.html>`_ for details)::
+
+	entry(
+	    index = 8,
+	    label = "N2H2 <=> NNH + H",
+	    degeneracy = 1,
+	    duplicate = True,
+	    kinetics = MultiPDepArrhenius(
+	        arrhenius = [
+	            PDepArrhenius(
+	                pressures = ([0.1, 1, 10], 'atm'),
+	                arrhenius = [
+	                    Arrhenius(A=(5.6e+36, '1/s'), n=-7.75, Ea=(70250.4, 'cal/mol'), T0=(1, 'K')),
+	                    Arrhenius(A=(1.8e+40, '1/s'), n=-8.41, Ea=(73390, 'cal/mol'), T0=(1, 'K')),
+	                    Arrhenius(A=(3.1e+41, '1/s'), n=-8.42, Ea=(76043, 'cal/mol'), T0=(1, 'K'))]),
+	            PDepArrhenius(
+	                pressures = ([0.1, 1, 10], 'atm'),
+	                arrhenius = [
+	                    Arrhenius(A=(1.6e+37, '1/s'), n=-7.94, Ea=(70757, 'cal/mol'), T0=(1, 'K')),
+	                    Arrhenius(A=(2.6e+40, '1/s'), n=-8.53, Ea=(72923, 'cal/mol'), T0=(1, 'K')),
+	                    Arrhenius(A=(1.3e+44, '1/s'), n=-9.22, Ea=(77076, 'cal/mol'), T0=(1, 'K'))])]))
+
+**Chebyshev**
+(see `Chebyshev Class <http://reactionmechanismgenerator.github.io/RMG-Py/reference/kinetics/chebyshev.html>`_ for details)::
+
+	entry(
+	    index = 9,
+	    label = "CH3 + OH <=> CH2(S) + H2O",
+	    degeneracy = 1,
+	    kinetics = Chebyshev(
+	        coeffs = [
+	            [12.4209, -0.799241, -0.299133, -0.0143012],
+	            [0.236291, 0.856853, 0.246313, -0.0463755],
+	            [-0.0827561, 0.0457236, 0.105699, 0.057531],
+	            [-0.049145, -0.0760609, -0.0214574, 0.0247001],
+	            [-0.00664556, -0.0412733, -0.0308561, -0.00959838],
+	            [0.0111919, -0.00649914, -0.0106088, -0.0137528],
+	        ],
+	        kunits='cm^3/(mol*s)', Tmin=(300, 'K'), Tmax=(3000, 'K'), Pmin=(0.0013156, 'atm'), Pmax=(131.56, 'atm')))
+
+
+The ``Troe``, ``Lindemann``, and ``Chebyshev`` pressure dependent formats could also be defined with a specific collider if needed, for example::
```

Comment:
Can you add a title so this doesn't get passed over as part of the Chebyshev section?